tests/test_webhook.py

```diff
@@ -176,6 +182,25 @@ def test_extract_media_info_fails(self):
         actual = helpers.extract_media_info(body)
         self.assertIsNone(actual)
 
+    def test_known_type_media(self):
+        """Check if the issue is already in the list of known type-media.
+
+        Scenarios:
+        1. the issue is already in the list with the same issue number.
+           (nothing to do. Might happen with change of labels.)
+           True
+        2. the issue is in the list with same error code and domain,
```

Comment:
(thanks, I reviewed these commits chronologically... and some things appeared a bit later on. ⭐️)